api/napi/index.ts

```diff
@@ -144,4 +145,75 @@ export class ArrayModel<T> implements Model<T> {
     entries(): IterableIterator<[number, T]> {
         return this.a.entries()
     }
+}
+
+export class Component {
+    private instance: ComponentInstance;
+
+    /**
+    * @hidden
+    */
+    constructor(instance: ComponentInstance) {
+        this.instance = instance;
+    }
+
+    run() {
+        this.instance.run();
+    }
+
+    show() {
+        this.instance.window().show();
+    }
+
+    hide() {
+        this.instance.window().hide();
+    }
+
+    get window(): Window {
+        return this.instance.window();
+    }
+}
+
+/**
+ * @hidden
+ */
+interface Callback {
+    (): any;
+    setHandler(cb: any): void;
+}
+
+export function create(path: string): Component | null {
```

Comment:
I think `create` is a name that's a little too "naked". It's like calling the function "new" :-). How about `createInstanceFromPath`?